character_generator/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+
+class User(models.Model):
```

Comment:
Delete this one, it already comes with Django (`django.contrib.auth.models.User`)